requirements.txt

```diff
@@ -1,4 +1,6 @@
 cryptography>=0.9.2
+enum34>=1.0.4
+ordereddict>=1.1
```

Comment:
In case you're wondering, we still install them, but it happens in setup.py, protected by an if-statement for Python 2.

The dot at the bottom of this file makes sure that we install boxsdk itself, and also that we run setup.py. That will pick up enum34 and ordereddict when necessary.

BTW, can you change the ``. ``at the bottom of this file to ``-e .``? Thanks.